ci/lint/stage_impls.py

```
@@ -543,6 +543,28 @@ def run_ruff() -> bool:
+    # increase fails. See ci/lint_python/subprocess_capture_checker.py.
+    result = subprocess.run(
+        [
+            "uv",
+            "run",
+            "python",
+            "ci/lint_python/subprocess_capture_checker.py",
+            "ci",
+            "test.py",
+            "--exclude",
+            "ci/tmp",
+        ],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)